doc/model/fluid.rst

```diff
@@ -0,0 +1,178 @@
+=====
+Fluid
+=====
+
+.. note::
+
+   This model document has been updated to be *MIRGE-Com* specific, and to include multi-component mixtures
+   with chemical reactions.
+
+.. important::
+
+   *MIRGE-Com* main repo branch currently provides fluid simulation operators and utilities for inviscid
+   flow (i.e. the :ref:`Euler equations<Euler-eqns>`).  Viscous flow simulation capabilities are forthcoming.
+
+.. raw:: latex
+
+    \def\RE{\operatorname{RE}}
+    \def\PR{\operatorname{PR}}
```

Comment:
Btw, if these are no longer used, you might as well drop them.

Reply:
(cc010df)